syntax = "proto3";

import "hq/dto/Security.proto";
import "hq/dto/OrderBook.proto";
option java_package = "com.huasheng.quant.open.sdk.protobuf.hq.response";
option java_outer_classname = "OrderBookResponseProto";
message OrderBookResponse {
    Security security = 1; //股票定义
	repeated OrderBook orderBookAskList = 2; //卖盘
	repeated OrderBook orderBookBidList = 3; //买盘
    double spreadLevel = 4; // 最小价格单位
}
